uploader.v2.json

```diff
@@ -615,6 +615,11 @@
             "pattern": [],
             "symbolic_name": "ovs_vsctl_show"
         },
+        {
+            "command": "/usr/sbin/gluster vol info",
+            "pattern": [],
+            "symbolic_name": "gluster vol info"
```

Comment:
the `symbolic_name` should be the same to the variable name in 'default.py' of insights-core, here, it should be `gluster_v_info`